public/pages/CreateTrigger/containers/CreateTrigger/CreateTrigger/CreateTrigger.js

```diff
@@ -316,46 +310,25 @@ export default class CreateTrigger extends Component {
                 <h1>{edit ? 'Edit' : 'Create'} trigger</h1>
               </EuiTitle>
               <EuiSpacer />
-              {isBucketLevelMonitor ? (
-                <FieldArray name={'triggerConditions'} validateOnChange={true}>
-                  {(arrayHelpers) => (
-                    <DefineBucketLevelTrigger
```

Comment:
I don't see `DefineBucketLevelTrigger` in the `DefineTriggerV1` file. Does this get used elsewhere now?